tests/unit/backend/chalice/api_server/test_v1_dataset.py

```diff
@@ -0,0 +1,60 @@
+import json
+import unittest
+
+import boto3
+from furl import furl
+from moto import mock_s3
+
+from tests.unit.backend.chalice.api_server import BaseAPITest
+
+
+class TestDataset(BaseAPITest, unittest.TestCase):
```

Comment:
Inherit `CorporaTestCaseUsingMockAWS` which already mocks out S3 and more and even has a built in function to create S3 objects.